models/users.js

```diff
@@ -540,6 +545,30 @@ const getUsersBasedOnFilter = async (query) => {
   return [];
 };
 
+const getUsersWithOnboardingState = async (query) => {
```

Comment:
can we not make it generic instead of strictly `ONBOARDING`

Reply:
tried fixing, have a look